pkg/cleanup/cleanup.go

```diff
@@ -73,6 +110,26 @@ func LimitTags(tags *[]string, keep int) []string {
 	return []string{}
 }
 
+// TagsOlderThan returns the tags which are older than the specified time
+func TagsOlderThan(imageStreamObjectTags *[]imagev1.NamedTagEventList, olderThan time.Time) []string {
```

Comment:
I propose to rename the method to `FilterImageTagsByTime` to align with `FilterByRegex`